doc/EMAIL.md

```diff
@@ -0,0 +1,10 @@
+# Sending Emails in Travel Support Program
+
+To send emails with travel support program you need to setup some external mail service. For example using the gmail smtp, just add the smtp settings using `config.action_mailer.smtp_settings` in the `environment` files.
+For gmail smtp settings you can checkout this <a href="https://support.google.com/a/answer/176600?hl=en">link</a>.
+
+There are various options provided for emails like `async_emails`, `email_from`, `email_footer`, `email_default_url_options` which can be set in the site.yml file.
```

Comment:
If this hasn't been documented yet, it would be good to do it now. Could you do that?